packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1641,11 +1641,11 @@ class RawEditorState extends EditorState
         return;
       }
 
-      if (!widget.scrollable) {
-        final localRect = renderEditor
-            .getLocalRectForCaret(textEditingValue.selection.extent);
-        renderEditor.showOnScreen(rect: localRect);
-      } else {
+      final localRect =
+          renderEditor.getLocalRectForCaret(textEditingValue.selection.extent);
+      renderEditor.showOnScreen(rect: localRect);
```

Comment:
Shouldn't we add the duration parameter here? (I omitted that in the previous PR)
```suggestion
      renderEditor.showOnScreen(
          rect: localRect,
          duration: withAnimation ? _caretAnimationDuration : Duration.zero);
```